n2y/notion_mocks.py

```diff
@@ -110,6 +110,15 @@ def mock_file(url):
     }
 
 
+def mock_link_to_page_block(linked_page_id):
```

Comment:
A `mock_link_to_page_block` function is probably not necessary, since you can just use `mock_block` with the appropriate values for the `content` argument. That being said, if you find yourself using this a lot and having a separate function for it simplifies the code, this would definitely be the place for it.

The function as written is incorrect though, since it should call `mock_block` to fill in all of the top-level properties, much like the `mmock_paragraph_block` does.

Reply:
Yes, I meant to clean that method out of there.  I realized some time ago that I didn't need a special mock.